cmd/client/main.go

```diff
@@ -151,6 +162,7 @@ func newGrpcProxyClientOptions() *GrpcProxyClientOptions {
 		requestPort:  8000,
 		proxyHost:    "localhost",
 		proxyPort:    8090,
```

Comment:
nit: Can you set the default proxy port to 0 if proxyUdsName!="", to save the user from explicitly setting --proxy-port=0?

Reply:
I like the idea but would rather we did it in a separate PR. We should restructure the start to have a defaulting layer for that.